src/utilities/logger.h

```diff
@@ -193,7 +193,7 @@ class LoggerManager
     static std::map<std::string, std::shared_ptr<LoggerManager>> loggers; //!< container for pointers to all the available contexts
     std::string name;  //!< context name
     std::shared_ptr<LoggingCore> loggingControl; //!< pointer to the actual Logger
-   LoggerManager(const std::string &loggingName);
```

Comment:
I think I would leave this one as a string reference since it is private and can only be called in on context and that context already has a string reference so this will just add a extra move operation,  which probably means I spent more time commenting on it it than it will ever spend in execution time.

Reply:
Sure

Reply:
Resolved, the deletion is changing the name `loggingName`->`loggerName`